openshift/kustomize/tekton/base/tasks/git-conditions.yaml

```diff
@@ -48,7 +51,10 @@ spec:
 
         # Update the git repo with the latest commit.
         git config --global --add safe.directory '*'
-        git checkout $(params.GIT_REF)
+        git fetch origin master
+        git fetch origin dev
+        git checkout $(params.GIT_SOURCE)
+        # TODO: Make the branch equal to the GIT_REF
```

Comment:
We're not using the GIT_REF currently in this step, but it would be nice to be able to.  The way it is currently implemented could result in pulling and using the most recent commit rather than the one that fired the webhook trigger.